### ISSUE TITLE
WORTMANN Computer support

### ISSUE THREAD

**Issue by gitdevmod** [InfotelGLPI/manufacturersimports#28]
Hi,

Is this possible to have support for WORTMANN Terra computer support. The url page is http://www.wortmann.de/en-gb/profile/snsearch.aspx only serial number is needed.

Thanks in advance

**Comment by gitdevmod** [comment]
Hi,

I just submitted a new PR #29 (not tested yet). This is my first attempt in plugin for glpi, I may did something wrong :)

**Comment by gitdevmod** [comment]
Plugin installs correctly. I can add manufacturer and search computer. In search result there is a link to view directly in website and it displays information but import do not works (import KO)
Thanks in advance for you help 

![capture da a cran_2016-11-26_00-49-58](https://cloud.githubusercontent.com/assets/2040775/20636268/15cc9474-b36a-11e6-83f4-334d5844be72.png)

**Comment by gitdevmod** [comment]
I made progress, I found en-gb version of the site fetch a pdf, I switched to fr-fr version and I can extract start and end date.

So far I do not know why I need to return "search" in getSearchField function. In result page it displays "à vie" and in computer page it displays 24 months.

![import](https://cloud.githubusercontent.com/assets/2040775/20644611/ae901ec6-b43a-11e6-8bdc-6602a1297a9e.png)
![result](https://cloud.githubusercontent.com/assets/2040775/20644612/b0f9f204-b43a-11e6-8bfb-dae70bd310f8.png)
